mgr/Conscript-standard

```diff
@@ -288,26 +288,28 @@ if ( $pkg !~ /^sim$/ && $pkg !~ /^gen$/ ) {
 	    $trgt = "";
 	    $hfl  = "";
 	    if ( $Dir =~ m/xgeometry/ ){
-		$fcheck = $trgt  = $OBJ."/".$Dir."/".$agM.".age";
-		$fcheck =~ s/\#//; $trgt = "" if ( -e $fcheck);
+		$trgt = "$OBJ/$Dir/$agM.age";
 	    } else {
 		# Compat contains redundant defs only needed for the age
 		# .h also needed for Dictionary building
 		if ( $xml !~ m/Compat/ ){
-		    $fcheck = $trgt  = $OBJ."/".$Dir."/".$agM.".cxx";
-		    $fcheck =~ s/\#//; $trgt = "" if ( -e $fcheck);
-
-		    $fcheck = $hfl   = $OBJ."/".$Dir."/".$agM.".h";
-		    $fcheck =~ s/\#//; $hfl = "" if ( -e $fcheck);
+		    $trgt = "$OBJ/$Dir/$agM.cxx";
+		    $hfl  = "$OBJ/$Dir/$agM.h";
 		}
 	    }
 	    # push but only if not present - if present, cons will find
 	    # them so there is no need for a predictive target rule
-	    push(@src, $trgt ) if ($trgt ne "" );
-	    push(@inc, $hfl)   if ($hfl  ne "" );
+	    push(@src, $trgt);
+	    push(@inc, $hfl);
```

Comment:
Are we sure about removing the condition? For example, `$hfl` will be empty for `$Dir = xgeometry`

Reply:
Sure, let's keep it